service/collector/vpc.go

```diff
@@ -0,0 +1,91 @@
+package collector
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	ClusterTag      = "giantswarm.io/cluster"
+	InstallationTag = "giantswarm.io/installation"
+	NameTag         = "Name"
+	OrganizationTag = "giantswarm.io/organization"
+	StackNameTag    = "aws:cloudformation:stack-name"
+
+	CidrLabel         = "cidr"
+	ClusterLabel      = "cluster_id"
+	IDLabel           = "id"
+	InstallationLabel = "installation"
+	NameLabel         = "name"
+	OrganizationLabel = "organization"
+	StackNameLabel    = "stack_name"
+	StateLabel        = "state"
+)
+
+var (
+	vpcs *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "vpc_info"),
+		"VPC information.",
+		[]string{
+			CidrLabel,
+			ClusterLabel,
+			IDLabel,
+			InstallationLabel,
+			NameLabel,
+			OrganizationLabel,
+			StackNameLabel,
+			StateLabel,
+		},
+		nil,
+	)
+)
+
+func (c *Collector) collectVPCs(ch chan<- prometheus.Metric) {
+	c.logger.Log("debug", "collecting metrics for vpcs")
+
+	resp, err := c.awsClients.EC2.DescribeVpcs(&ec2.DescribeVpcsInput{})
```

Comment:
Some caching would actually be cool so we fetch the information all the time and have a prepared bucket of information we can just send out all the time.